docs/Documentation/Environment/lmod.md

```diff
@@ -0,0 +1,149 @@
+
+Our production location for module examples is `/nopt/nrel/apps/modules/default` on Kestrel and `/nopt/nrel/apps/modules/default` on Eagle. 
+This path is set for everyone by default. 
+In addition, motivated users who would like to support an application for the *"energy community"* on Eagle may request write access to the *"e-com"* space. 
+Modules there are located in `/nopt/nrel/ecom/modulefiles`, and are usable by all Eagle users (see the module use command under *"Common Module Commands"* section below).
+
+
+## Commom Module Commands 
+
+The `module` command accepts parameters that enable users to inquire about and change the module environment. 
+Most of the basic functionality can be accessed through the following commands.
+
+| option  | description |
+|:---------------------:|:------------|
+|`spider`	                |Prints available modules in a path-agnostic format.|
+|`avail`	                |Prints available modules grouped by path. |
+|`list`	                    |Prints all currently loaded modules.|
+|`display 'name'`	        |Prints settings and paths specified for a particular module.|
+|`help 'name'`	            |Prints help message for a particular module.|
+|`load 'name'`	            |Loads particular module. For modules listed as the '(default)', the short package name is sufficient. To load another version of the package the long package name is required (e.g., `module load fftw/3.3.10-cray-mpich-intel`).|
+|`unload 'name'`	        |Unloads particular module.|
+|`swap 'name1' 'name2'`	|First unload `modName1` and then load `modName2`.|
+|`use {-a} A_PATH`	    |Prefix `{suffix}` the path `$A_PATH` to your `$MODULEPATH` variable, in order to find modules in that location.|
+|`unuse A_PATH`	        |Remove the path `$A_PATH` from your `$MODULEPATH` variable.|
+
+## Module Organization on NREL HPC 
+
+The modulefiles that we provide are only a starting point. 
+For maximum control, users should copy these files from the locations in `/nopt` to their own locations for which they have write access.
+
+Module files for baseline *applications*, *libraries*, *frameworks*, and *toolchains* are located in: 
+|HPC | Modulefies Location |
+|:---:|:-----|
+|Kestrel | `/nopt/nrel/apps/modules/default ` |
+|Eagle | `/nopt/nrel/apps/modules/default/modulefiles/ ` |
+
+Note: 
+It should be mentioned that for Kestrel the modulefiles are organized under 3 categories
+
+1-  Compilers and Mpis 
+2-  Utilities and Libraries 
+3-  Applications 
+
+Users may and should freely copy these example modulefiles to preferred locations and customize them for their own use cases. 
+This can be particularly desirable to preserve a critical workflow as the software environment changes on Kestrel and Eagle, or to change the behavior, e.g., turn off automatic loading of prerequisites. 
```

Comment:
```suggestion
This can be particularly desirable to preserve a critical workflow as the software environment is not the same between systems, or to change the behavior, e.g., turn off automatic loading of prerequisites. 
```
Or something to similar to make it general